OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.java

```diff
@@ -62,13 +79,35 @@
     @Inject
     private LoginPrefs loginPrefs;
 
+    @Inject
+    private UserService userService;
+
     private FullScreenErrorNotification errorNotification;
 
     private SnackbarErrorNotification snackbarErrorNotification;
 
     // Reason of usage: Helps in deciding if we want to show a full screen error or a SnackBar.
     private boolean isInitialServerCallDone = false;
 
+    private ProfileModel profile;
+
+    private ToolbarCallbacks toolbarCallbacks;
+
+    @Nullable
+    private Call<Account> getAccountCall;
+
+    /**
+     * The container Activity must implement this interface so the frag can communicate
+     */
+    public interface ToolbarCallbacks {
```

Comment:
All these callbacks aren't essentially part of `MyCoursesListFragment` and should actually reside in the Tabs container fragment that's being implemented in #1048.

You can add a `TODO` here to move all this code to the relevant fragments once #1048 is merged.